api/src/home/analysis-platform/data/AnalysisPlatformDS/Blueprints/SIMAApplicationInput.json

```diff
@@ -0,0 +1,43 @@
+{
+  "name": "SIMAApplicationInput",
+  "type": "system/SIMOS/Blueprint",
+  "extends": [
+    "system/SIMOS/DefaultUiRecipes",
+    "system/SIMOS/NamedEntity"
+  ],
+  "description": "Input to container",
+    "attributes": [
+    {
+      "type": "system/SIMOS/BlueprintAttribute",
+      "name": "inputType",
+      "description": "input type",
+      "attributeType": "string"
+    },
+    {
+      "type": "system/SIMOS/BlueprintAttribute",
+      "name": "outputType",
+      "description": "output type",
+      "attributeType": "string"
+    },
+    {
+      "type": "system/SIMOS/BlueprintAttribute",
+      "name": "input",
+      "description": "Input entity to container as json string (must be of same type as defined by inputType)",
+      "attributeType": "string"
+    },
+    {
+      "name": "SIMAComputeConfig",
+      "attributeType": "system/SIMOS/Blob",
+      "type": "system/SIMOS/BlueprintAttribute",
+      "description": "",
+      "optional": true
+    },
+    {
+      "name": "applicationConfig",
+      "attributeType": "AnalysisPlatformDS/Blueprints/SIMAApplicationConfig",
+      "type": "system/SIMOS/BlueprintAttribute",
+      "description": "Extra configs",
+      "optional": true
+    }
```

Comment:
Think this should be flattened. As it is now optional, and will make things a bit easier